indico/modules/events/registration/templates/management/reglist_filter.html

```diff
@@ -2,29 +2,20 @@
 
 {% macro _render_regform_item_col_selector(section) %}
     <h3>{{ section.title }}</h3>
-    {% for child_item in section.children if child_item.is_field %}
-        {% set has_dropdown = child_item.input_type in ['single_choice', 'country', 'bool', 'checkbox'] %}
-        <div class="i-label reglist-column js-regform-item {%- if has_dropdown %} filter{% endif %}"
+    {% for child_item in section.children if child_item.is_field and not child_item.is_deleted %}
+        <div class="i-label reglist-column js-regform-item {%- if child_item.field_impl.filter_choices %} filter{% endif %}"
```

Comment:
you use `filter_choices` quite often. why not `{% set filter_choices = ... %}` to avoid calling the property over and over?